backend/backend/settings.py

```diff
@@ -62,6 +62,7 @@
 INSTALLED_APPS = DJANGO_APPS + THIRD_PARTY_APPS + LOCAL_APPS
 
 MIDDLEWARE = [
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
The PR adds 'corsheaders.middleware.CorsMiddleware' to MIDDLEWARE but does not add 'corsheaders' to INSTALLED_APPS, nor any CORS settings. If django-cors-headers is not installed, startup will fail; and omitting INSTALLED_APPS/config may produce incomplete behavior.

Either remove this change, or add instructions and config: add 'corsheaders' to INSTALLED_APPS and configure allowed origins (e.g. CORS_ALLOWED_ORIGINS or CORS_ALLOW_ALL_ORIGINS), and ensure requirements.txt/Pipfile includes django-cors-headers.